Modules/ITS/include/ITS/ITSTrackTask.h

```diff
@@ -63,24 +74,24 @@ class ITSTrackTask : public TaskInterface
   std::vector<TObject*> mPublishedObjects;
   TH1D* hNClusters;
   TH1D* hNClustersReset;
-  TH1D* hTrackEta;
-  TH1D* hTrackPhi;
+  std::unique_ptr<TH1DRatio> hTrackEta;
+  std::unique_ptr<TH1DRatio> hTrackPhi;
   TH1D* hVerticesRof;
-  TH2D* hAngularDistribution;
+  std::unique_ptr<TH2DRatio> hAngularDistribution;
   TH2D* hVertexCoordinates;
   TH2D* hVertexRvsZ;
   TH1D* hVertexZ;
   TH1D* hVertexContributors;
   TH1D* hAssociatedClusterFraction;
   TH1D* hNtracks;
   TH1D* hNtracksReset;
-  TH2D* hNClustersPerTrackEta;
-  TH2D* hNClustersPerTrackPhi;
-  TH2D* hHitFirstLayerPhiAll;
-  TH2D* hHitFirstLayerPhi4cls;
-  TH2D* hHitFirstLayerPhi5cls;
-  TH2D* hHitFirstLayerPhi6cls;
-  TH2D* hHitFirstLayerPhi7cls;
+  std::unique_ptr<TH2DRatio> hNClustersPerTrackEta;
+  std::unique_ptr<TH2DRatio> hNClustersPerTrackPhi;
+  std::unique_ptr<TH2DRatio> hHitFirstLayerPhiAll;
+  std::unique_ptr<TH2DRatio> hHitFirstLayerPhi4cls;
+  std::unique_ptr<TH2DRatio> hHitFirstLayerPhi5cls;
+  std::unique_ptr<TH2DRatio> hHitFirstLayerPhi6cls;
+  std::unique_ptr<TH2DRatio> hHitFirstLayerPhi7cls;
   TH2D* hClusterVsBunchCrossing;
   TH2D* hNClusterVsChipITS;
```

Comment:
I think also `hNClusterVsChipITS` needs the same treatment. In general all the plots with the `kIsAverage` bit set. Maybe give another rapid look at the two tasks to be sure you did not miss any plot, thank you!

Reply:
Yes, indeed, I didn't implement TH2Ratio for `hNClusterVsChipITS `. The problem is that this object has non-uniform binning defined by an array, and it is impossible to construct the THRatio object in such a way. We need to change the mother class to have  `hNClusterVsChipITS ` scaled correctly. Otherwise, we can keep this object as it is.

Reply:
ok good point, we could keep the object as it is. In the end we are not interested in the absolute value of the z axis.